sdk/identity/identity/src/credentials/interactiveBrowserCredential.browser.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import * as msal from "msal";
+import * as msalBrowser from "@azure/msal-browser";
+import * as msalCommon from "@azure/msal-common";
```

Comment:
you shouldn't need to import msal-common, the types are all re-exported from msal-browser. I recommend only using msal-browser so you can limit the number of deps you have.

Reply:
Thank you! I'll do that 🙏